alerts/resource_alerts.libsonnet

```diff
@@ -72,6 +72,32 @@
             },
             'for': '5m',
           },
+          {
+            alert: 'NodeCPUUsage',
+            annotations: {
+              message: 'Node {{ $labels.node }} has CPU usage {{ $value }} percent (which is above the configured threshold of %s percent).' % ($._config.maxNodeCpuUsagePercent),
+            },
+            expr: |||
+              (node:node_cpu_utilisation:avg1m * 100) > %(maxNodeCpuUsagePercent)s
+            ||| % $._config,
+            'for': '5m',
```

Comment:
5m is way to trigger happy. If we want this at all (see my general comment on the PR) this should be at least 1h.